src/itkConfig.ts

```diff
@@ -1,4 +1,4 @@
-const version = '1.0.0-b.18'
+import { version } from '.'
```

Comment:
I think this should be `./index.js`. The JavaScript/TypeScript build is failing on this line.